File: 2219-maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        max = 0;
        for sentence in sentences:
            count = 0;
            for word in sentence:
                if word == " ":
                    count += 1;
            if count > max:
                max = count;
        return max+1;

File: 2219-maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.go
func maxCount(a, b int) int{
    if a >= b{
        return a;
    }
    return b;
}

func mostWordsFound(sentences []string) int {
    max := 0;
    for _, words := range sentences{
        count := len(strings.Split(words, " "));
        max = maxCount(max, count); 
    }
    return max;
}